docs/src/pages/ui/primitives/menu/react.mdx

```diff
@@ -0,0 +1,224 @@
+import { Menu, MenuItem } from '@aws-amplify/ui-react';
+
+import { MenuDemo } from './demo';
+import {
+  AlignExample,
+  BasicExample,
+  MenuExample,
+  ControlledExample,
+  SizeExample,
+} from './examples/';
+
+import { Example } from '@/components/Example';
+
+Menu provides an accessible, interactive menu for selecting actions within an application. Dropdown menu is collision-aware and will automatically change location based on available space.
+
+## Demo
+
+<MenuDemo />
+
+## Usage
+
+Import the `Menu` and `MenuItem` components and styles. Use `MenuItem` component to configure menu options.
+
+```tsx file=./examples/basicExample.tsx
+
+```
+
+<Example>
+  <BasicExample />
+</Example>
+
+### Size
+
+Control the size of the Menu buttons and items using the `size` prop. Available options are `small`, none (default), and `large`.
+
+```tsx file=./examples/sizeExample.tsx
+
+```
+
+<Example>
+  <SizeExample />
+</Example>
+
+### Align
+
+Alignment of menu with button trigger can be controlled using the `align` prop. Available options are `start` (default), `center`, and `end`.
+
+```tsx file=./examples/alignExample.tsx
+
+```
+
+<Example>
+  <AlignExample />
+</Example>
+
+### Customize trigger button
+
+The default trigger button can be overwritten using the `trigger` prop on `Menu` with the `MenuButton` component.
+`MenuButton` can take all the same props as [Button](./button).
+
+```tsx file=./examples/menuExample.tsx
+
+```
+
+<Example>
+  <MenuExample />
+</Example>
+
+### Controlled Menu
+
+Create a controlled `Menu` using the `isOpen` and `onOpenChange` props.
+
+```tsx file=./examples/controlledExample.tsx
+
+```
+
+<Example>
+  <ControlledExample />
+</Example>
+
+## CSS Styling
+
+### Target classes
```

Comment:
I love this! Is there any way we can automate this from the ComponentClassNames or something?

Reply:
I was wondering that as well. Maybe we could grab all the ComponentClassNames that start with the `Menu` prefix...